AUTHORS.md

```diff
@@ -14,7 +14,6 @@
 
 -   Alexandre Catarino([@AlexCatarino](https://github.com/AlexCatarino))
 -   Arvid JB ([@ArvidJB](https://github.com/ArvidJB))
--   Benoît Hudson ([@benoithudson](https://github.com/benoithudson))
```

Comment:
HM, I wonder how these guys were removed?

Reply:
This is probably just due to the QuantConnect fork missing a few changes (in this case the domain unload fixes).

Reply:
@jaredbroad can you cherry-pick the domain unload fixes from master onto your branch?